apps/storefront/src/pages/PDP/useAddedToShoppingListAlert.tsx

```diff
@@ -1,8 +1,32 @@
+import { ReactNode } from 'react';
 import { useB3Lang } from '@b3/lang';
 import { Box, Button } from '@mui/material';
 
 import { useAppSelector } from '@/store';
-import { globalSnackbar } from '@/utils';
+import { globalSnackbar, platform } from '@/utils';
+
+function CatalystButton({
+  goToShoppingDetail,
+  children,
+}: {
+  goToShoppingDetail: () => void | undefined;
```

Comment:
other than the onClick naming, this is the same as `B3AddToQuoteTip.tsx`

which again makes me wonder about needing a more robust system for picking the variants